### PR TITLE
Articles component

### PR DESCRIPTION
**Pull request by Ebenezr**
image

### CODE REVIEW

File: src/components/Articles/BlogCards.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  id?: number;
+  badge?: string;
+  imgUrl?: string;
+  title?: string;
+  writer?: string;
+  timestamp?: string;
+  videoUrl?: string;
+}
+
+const BlogCards: React.FC = () => {
```
Comment (nyandika) [inline src/components/Articles/BlogCards.tsx:15]: This is not a recommended way of creating components. it's been removed from template actually:  [Link](https://github.com/facebook/create-react-app/pull/8177)

Comment (nyandika) [reply src/components/Articles/BlogCards.tsx:15]: you can use: `const Blogcard = () => {}`

File: src/components/Articles/BlogCards.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  const [articles, setArticles] = useState<articlesProp[]>([]);
+  // get request to get articles
+  const getArticles = async () => {
+    return await Axios.get('/articles');
+  };
+  // onmount hydrate articles state
+  useEffect(() => {
+    getArticles().then((response) => setArticles(response?.data));
```
Comment (nyandika) [inline src/components/Articles/BlogCards.tsx:23]: You can handle all fetching and parsing in the `getArticles` method and just get the data here

**Review by nyandika — CHANGES_REQUESTED** [review]
You should also commit the package-lock file

I also noticed you deleted the other PR to create a new one, you dont have to do this. making changes on the same branch and pushing to the same branch will automatically update the PR. let me know if you need further help on that and we can setup a small sync

**Comment by Ebenezr** [comment]
noted

File: src/components/Articles/BlogCards.tsx (hunk 2 above)
Comment (nyandika) [reply src/components/Articles/BlogCards.tsx:23]: Yea this is perfect

**Review by nyandika — APPROVED** [review]
There are a couple of nits that we could refactor as we go but this is good progress

**Comment by nyandika** [comment]
I pushed some updates to lib dependencies which had led to conflicts on this branch. Let me know if you need help resolving this. I can set up a sync

**Comment by Ebenezr** [comment]
image

I have fixed the conflicts, but I'm version error when I try to npm install, 
NB: I have updated my node and npm to the latest version already.

**Pull request closed without merging**